ballerina/tests/test.bal

```diff
@@ -0,0 +1,281 @@
+        sharePermissions: [],
+        editPermissions: []
+    });
+    test:assertNotEquals(response.id, null, "Dashboard id should not be empty.");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetDefaultShareScope() returns error? {
+    // Call the get default share scope function
+    DefaultShareScope response = check jira->/rest/api/'3/filter/defaultShareScope();
+
+    // Assertion: Check that 'scopeType' exists in the response
+    test:assertNotEquals(response.scope, "", "Expected 'scope' to have a value");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testPutDefaultShareScope() returns error? {
+    // Call the put function
+    DefaultShareScope response = check jira->/rest/api/'3/filter/defaultShareScope.put(payload = {scope:"AUTHENTICATED" });
+
+    // Assertion: Check that the response contains the expected scope value
+    
+    test:assertEquals(response.scope, "AUTHENTICATED", msg = "Expected 'scope' to be 'AUTHENTICATED'");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetFavouriteFilters() returns error? {
+
+  Filter[] response = check jira->/rest/api/'3/filter/my();
+
+  test:assertNotEquals(response.length(), 0, "There are no My Filters.");
+
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetMyFilters() returns error? {
+
+    Filter[] response = check jira->/rest/api/'3/filter/my();
+
+    test:assertNotEquals(response.length(), 0, "There are no My Filters.");
+
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testPostGroup() returns error? {
+
+    // Call the function to create a group
+    Group response = check jira->/rest/api/'3/group.post(payload = {name: "first group"});
+
+    //Assertation
+    test:assertEquals(response.name, "first group", "Group name does not match the expected value.");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testDeleteGroup() returns error? {
+
+    // Call the function to delete the specified group
+    http:Response response = check jira->/rest/api/'3/group.delete(queries = {groupname: "first group"});
+    // Validate the response
+    test:assertEquals(response.statusCode, 200, "Expected HTTP status code 200 indicating successful deletion.");
+
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetAllFieldConfigurations() returns error? {
+
+    PageBeanFieldConfigurationDetails response = check jira->/rest/api/'3/fieldconfiguration();
+
+    test:assertTrue(response.total > 0, "There is no field configuration");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetAllFields() returns error? {
+
+   FieldDetails[] response = check jira->/rest/api/'3/field();
+
+   test:assertNotEquals(response.length(), 0, "Expected field details array to be non-empty.");
+
+}
```

Comment:
Add a new line
```suggestion
}

```

Reply:
@Bhashinee, Hey I'm really sorry that I have not tried building the code yesterday and for some strange reason, VS code also did not showed any errors. I have fixed everything now, ran the tests locally and it worked